types/nightwatch/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
do you folks need this file? If yes, what's the reason?

Reply:
Even though there is no violation of the rules mentioned in `eslintrc`, I was still getting errors in ESlint. For instance, there is no `const enum` declaration in the file, but ESlint is still complaining it

Reply:
If you have any suggestions, then I am happy to remove this file and incorporate those changes.

Reply:
I think you're reffering to failing CI builds. That was amended already. Those types should work fine without any additional configuration.

Reply:
Thanks, I removed it, and the build is now passing.

Reply:
@peterblazejewicz Can you also help me fix this issue:
![image](https://user-images.githubusercontent.com/8705386/180970242-6d5d9903-4e0b-4c6d-9c95-124e0b6d3239.png)

What have I done so far?
- Did NPM install
- Install `@typescript-eslint/parser`

Reply:
I am self-merging since I have resolved the main issue